cogs/birthday/birthday.py

```diff
@@ -41,10 +39,22 @@ def __init__(self, bot: Red):
             )
             self.logger.addHandler(handler)
 
+    def initializeBgTask(self):
         # On cog load, we want the loop to run once.
         self.lastChecked = datetime.now() - timedelta(days=1)
         self.bgTask = self.bot.loop.create_task(self.birthdayLoop())
 
+    # Class constructor
+    def __init__(self, bot: Red):
+        self.bot = bot
+        self.bgTask: asyncio.Task = None
+        self.config = None
+        self.lastChecked = None
```

Comment:
Might as well add type hints for these vars.
```suggestion
        self.config: Config = None
        self.lastChecked: datetime = None
```